libr/bin/p/symbols.mk

```diff
@@ -7,4 +8,5 @@ ALL_TARGETS+=${TARGET_SYMBOLS}
 
 ${TARGET_SYMBOLS}: ${OBJ_SYMBOLS}
 	-${CC} $(call libname,bin_mach0) ${CFLAGS} \
-		${OBJ_SYMBOLS} $(LINK) $(LDFLAGS) 
+		${OBJ_SYMBOLS} ${SHLR}/sdb/src/libsdb.a \
```

Comment:
Uhm i think this shuold be added if u include sdb/deps.mk

Reply:
apparently there's no such `shlr/sdb/deps.mk`

Reply:
:_